omero-server/templates/ingress.yaml

```
@@ -21,6 +21,9 @@ metadata:
 {{ toYaml . | indent 4 }}
 {{- end }}
 spec:
+{{- if .Values.ingress.className }}
+  ingressClassName: {{ .Values.ingress.className }}
```

```suggestion
{{- with .Values.ingress.className }}
  ingressClassName: {{ . }}
```
Just a minor style point- this avoids some repetition